src/js/firstDiscoveryEditor.js

```diff
@@ -23,9 +23,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         gradeNames: ["fluid.viewRelayComponent", "fluid.prefs.prefsEditorLoader", "autoInit"],
         selectors: {
             prefsEditor: ".gpiic-prefsEditor",
-            help: ".gpiic-help"
```

Comment:
Because I removed the "help" button from my test index page since there's no actual implementation for it at this point. The proper selectors will be added in another branch that has Jon and Anastasia's real template @ https://github.com/fluid-project/first-discovery/pull/3 merged in.